api/src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_eac_data/create_venues.py

```diff
@@ -36,7 +36,7 @@ def create_venues(offerer_list: list[offerers_models.Offerer]) -> None:
         departementCode="56",
         postalCode="56000",
         city="Lorient",
-        siret="55208131766522",
+        siret=f"{offerer.siren}66522",
```

Comment:
Es-tu sûr que cela crée des SIRET valides suivant l'algorithme de Luhn ?
Proposition pour coller à quelque chose de plus réaliste : 
```suggestion
        siret=siren_utils.complete_siren_or_siret("{offerer.siren}0001"),
```